pkg/network/encoding/marshal/format.go

```diff
@@ -97,7 +97,16 @@ func FormatConnection(builder *model.ConnectionBuilder, conn network.ConnectionS
 		dynamicTags map[string]struct{}
 	)
 	staticTags, dynamicTags = httpEncoder.GetHTTPAggregationsAndTags(conn, builder)
-	_, _ = http2Encoder.WriteHTTP2AggregationsAndTags(conn, builder)
+	staticTags2, dynamicTags2 := http2Encoder.WriteHTTP2AggregationsAndTags(conn, builder)
```

Comment:
1. let's rename the variable
2. consider creating functions `mergeStaticTags(tags ...uint64) uint64` and `mergeDynamicTags(dynamicTags ...map[string]struct{}) map[string]struct{}` make the code to better support future changes (like kafka tags)

Reply:
I've renamed the variables and added a function to merge the dynamic tags. For the static tags, I've left the merging to be done inline since it's just a bitwise OR.